cql2pgjson/src/test/java/org/folio/cql2pgjson/CompoundIndexTest.java

```diff
@@ -11,7 +11,7 @@ public void multiFieldNamesGIN() throws Exception {
     CQL2PgJSON cql2pgJson = new CQL2PgJSON("tablea");
     cql2pgJson.setDbSchemaPath("templates/db_scripts/compoundIndex.json");
     String sql = cql2pgJson.toSql("fullname == \"John Smith\"").toString();
-    String expected = "WHERE lower(concat_space_sql(tablea.jsonb->>'firstName' , tablea.jsonb->>'lastName')) LIKE lower('John Smith')";
+    String expected = "WHERE CASE WHEN length('John Smith') <= 600 THEN left(lower(concat_space_sql(tablea.jsonb->>'firstName' , tablea.jsonb->>'lastName')),600) LIKE left(lower('John Smith'),600)  ELSE left(tablea->>'fullname',600)  LIKE left('John Smith',600)  AND tablea->>'fullname'  LIKE  'John Smith' END";
```

Comment:
Currently compoundIndex.json doesn't have any b-tree indexes. The length case clause should only be used if there exist a b-tree index.

Reply:
A test case for a b-tree index on a compound field should be added, though.

Reply:
see comment below, but I do not understand what needs to be done here to change this.